plugin/cmd/kubectl/binding/binding.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kubernetes-incubator/service-catalog/plugin/pkg/kubectl/client"
+	"github.com/kubernetes-incubator/service-catalog/plugin/pkg/kubectl/utils"
+)
+
+const usage = `Usage:
+  kubectl plugin binding SUBCOMMAND
+
+Available subcommands:
+  list
+  get
+`
+
+const listUsage = `Usage:
+  kubectl plugin binding list NAMESPACE
+`
+
+const getUsage = `Usage:
+  kubectl plugin binding get NAMESPACE INSTANCENAME
```

Comment:
Why not `-n, --namespace=` instead of an arg?

Reply:
I think it depends on whether its a required arg or not. As of now its required. I know some other kubectl commands use `--arg` for required fields, but personally, I find that very odd.

Having said all of that, I think we do need to look ahead and think about how we're going to deal with cluster and NS scoped resources. That's when an optional `--namespace` might come into play.

Reply:
Even if it's required for the processing, it should not be required for users *to provide*. In `kubectl` every command that is namespaced defaults to the current "in use" namespace if you don't provide it explicitly, and plugins should do the same for consistency. 

The `KUBECTL_PLUGINS_CURRENT_NAMESPACE` will give you the namespace you must consider in the call, ready to use, with all the processing related to config file and env var precedence processed.

Reply:
Ah good point - I forgot that kubectl has the notion of the current/default namespace. I would,however, prefer to use kubectl's and not create a new plugin-specific current/default namespace. To me plugins are extending kubectl, not an alternative to it, so whatever defaults kubectl has should apply to plugins as well.

Reply:
Exactly! So just use the value in `KUBECTL_PLUGINS_CURRENT_NAMESPACE` and you get what `kubectl` considered as the namespace for the current call, with defaults, flags and env vars already processed.

Reply:
I'm confused. Isn't `KUBECTL_PLUGINS_CURRENT_NAMESPACE` a plugin thing instead of the default kubectl namespace? Shouldn't we grab it from the current context or something like that?

Reply:
It is what plugins must consider as the namespace to use in the current call. If you need a namespace, just use the value set in that env var *as is* and you're good.

kubectl does all its namespace processing (take the default namespace, kubeconfig, what user provided through `-n`, etc), processes the precedence logic, and sets the result in that env var. This way plugin writers don't have to duplicate all this logic.

Reply:
I'm not suggesting we duplicate it, we should just use it. Part of what @jberkhahn is doing is creating a shared library kind of thing and I would think the getting the current NS would be a util in there that both kubectl and plugins can leverage.  IOW, the "current namespace" should yield the same answer for kubectl as it does for the plugin.